requirements-base.txt

```diff
@@ -1,61 +1,118 @@
-alabaster==0.7.12
-backports.functools-lru-cache==1.6.1
-backports.ssl-match-hostname==3.7.0.1
-beautifulsoup4==4.9.1
-cffi==1.14.3
-click==7.1.2
-contextlib2==0.6.0.post1
-cryptography==3.3.2
+# These requirements are missing the psycopg2 package whose installation
+# is complicated by having two versions (psycopg2-binary and psycopg2)
+# See https://github.com/psycopg/psycopg2/issues/674
+# and https://www.psycopg.org/articles/2018/02/08/psycopg-274-released/
+# 
+# This file lists all the requirements except for the psycopg2 requirement
+# which is specified separately in requirements.txt and requirements-dev.txt
+# 
+Babel==2.8.0
+boto3==1.23.2
+botocore==1.26.2
+certifi==2021.10.8
+cffi==1.15.0
+chardet==4.0.0
+click==8.1.3
+contextlib2==21.6.0
+cryptography==37.0.2
 # defusedxml is required for SAML authentication
-defusedxml==0.6.0
-dm.xmlsec.binding==2.1
```

Comment:
I don't think `dm.xmlsec.binding` is required now with `python3-saml`. I think it was required `python-saml`. If that's not the case though then this will need to wait until support for versions of Python > 3.6 is pushed to PyPI.

Reply:
That's really good news, because these two packages were the reason we couldn't upgrade Python earlier.